addons/source-python/packages/source-python/translations/strings.py

```diff
@@ -217,12 +217,6 @@ def _replace_escaped_sequences(given_string):
         # Return the replaced string
         return given_string
 
-    def get_strings(self, key, **tokens):
```

Comment:
Okay, now I see where it comes from. But in my opinion, we shouldn't tokenize the original `TranslationStrings` instances that are stored in `LangStrings`. How about this:
```python
def get_strings(self, key, **tokens):
    """Return a new TranslationStrings object with updated tokens."""
    strings = TranslationStrings()
    strings.update(self[key])
    strings.tokens.update(tokens)
    return strings
```